test/datahike/test/utils.cljc

```diff
@@ -1,7 +1,50 @@
 (ns datahike.test.utils
-  (:require [datahike.api :as d]))
+  (:require [datahike.api :as d])
+  (:import (java.util UUID)))
 
-(defn setup-db [cfg]
-  (d/delete-database cfg)
-  (d/create-database cfg)
-  (d/connect cfg))
+(defn deep-merge
```

Comment:
We already have another `deep-merge` in the [config ns](https://github.com/replikativ/datahike/blob/7e60af807dd4db1d0eb73b75ac2f010e31361a3a/src/datahike/config.cljc#L85), but this here looks more sophisticated. Maybe we could move this to `datahike.tools` and use it in `datahike.config`?

Reply:
good point :t-rex: